recipes/markdown-preview-mode

```diff
@@ -1 +1,4 @@
-(markdown-preview-mode :fetcher github :repo "ancane/markdown-preview-mode")
+(markdown-preview-mode
+  :fetcher github
+  :repo "ancane/markdown-preview-mode"
+  :files ("*.el" "preview.html"))
```

Comment:
I suppose you should use `:defaults` instead of `"*.el"`.

See also
- https://github.com/milkypostman/melpa#recipe-format